src/macros.jl

```diff
@@ -965,18 +965,22 @@ function parsevariable(_error::Function, infoexpr::VariableInfoExpr, sense::Symb
     # Variable declaration of the form: var sense value
```

Comment:
This comment is out of date, so are the argument names.

Reply:
I explains where the function arguments come from. We should probably rename to lhs rhs though

Reply:
I have refactored it in two methods